moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -109,44 +114,39 @@ const useColumns = ({
   handleSaveClick,
   handleCancelClick,
   handleEditClick,
+  deptUnitData,
 }) =>
   useMemo(() => {
     return [
       {
         headerName: "Source",
-        field: "funding_source_id",
+        field: "moped_fund_source",
         width: 200,
         editable: true,
-        renderCell: ({ value }) =>
-          getLookupValueByID(
-            data["moped_fund_sources"],
-            "funding_source",
-            value
-          ),
+        valueFormatter: (value) => value?.funding_source_name,
         renderEditCell: (props) => (
           <LookupAutocompleteComponent
             {...props}
             name={"funding_source"}
-            lookupTable={data["moped_fund_sources"]}
+            options={data["moped_fund_sources"]}
+            fullWidthPopper={true}
+            autocompleteProps={{}}
```

Comment:
Another way to handle this could be a default value for the prop in the `LookupAutocompleteComponent` or use optional chaining when reach the prop values in the component code. Caught my eye and had me thinking about how to avoid passing an empty object.